.github/workflows/switch.yml

```diff
@@ -0,0 +1,125 @@
+name: Switch workflow branches
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version to check"
+        required: false
+    workflow_run: 
+      workflows: [Notify new upstream release]
+      types: [completed]
```

Comment:
`Notify new upstream release` will always succeed and it runs every day, this action should only run when a new upstream branch is created, I was thinking calling this action [like this](https://github.com/gitpod-io/openvscode-releases/blob/41232a840ca81f4101c7738ebd645cd97683df0d/.github/workflows/release.yml#L272-L282)

Reply:
thanks @jeanp413, changed it to work like that in [`94a4531` (#23)](https://github.com/gitpod-io/openvscode-releases/pull/23/commits/94a4531de183e1a8695849c4b2ca6ee046da8141).